slurm/03_casestudies_only.sh

```
@@ -0,0 +1,63 @@
+# Run ONLY case study analyses (real data)
+# Array: 108 jobs = 3 diseases × 6 gt_levels × 6 inc_levels
+# GT and INC vary independently
+# Limited to 40 concurrent (LSHTM limit)
+#
+# Jobs 1-36:   ebola   (gt 1-6 × inc 1-6)
+# Jobs 37-72:  covid   (gt 1-6 × inc 1-6)
+# Jobs 73-108: cholera (gt 1-6 × inc 1-6)
+
+# Get project root from SLURM_SUBMIT_DIR
+if [[ "$SLURM_SUBMIT_DIR" == */slurm ]]; then
+    cd "$SLURM_SUBMIT_DIR/.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)